src/pricecypher/endpoints/datasets.py

```diff
@@ -0,0 +1,108 @@
+from pricecypher.endpoints.base_endpoint import BaseEndpoint
+from pricecypher.models import Scope, ScopeValue, TransactionSummary, Transaction
+from pricecypher.rest import RestClient
+
+
+class DatasetsEndpoint(BaseEndpoint):
+    """PriceCypher dataset endpoints in dataset service.
+
+    :param str bearer_token: Bearer token for PriceCypher (logical) API. Needs 'read:datasets' scope.
```

Comment:
Like? The package currently does not have a way to query business cells, so that scope is not required. Same for configuration, price capping, and user settings